src/util/Utilities.js

```diff
@@ -1,3 +1,6 @@
+import config from 'react-global-configuration';
+var ACCOUNTS, web3;
+
 class Utilities {
```

Comment:
not sure how this works, but is an empty class required?

Reply:
not really required but useful. in the future we will have a constructor in there and at least on or two variable that we can use through out different components. for now at least im exporting one default of this calls otherwise react throws errors